gnovm/pkg/gnolang/preprocess.go

```diff
@@ -2352,6 +2352,8 @@ func preprocess1(store Store, ctx BlockNode, n Node) Node {
 						// }
 						*dst = *dt2
 					}
+				case PrimitiveType:
```

Comment:
WDYT, handle other types like PointerType, NativeType ?
example this:

```
package main

type (
	Number  = *int32
	Number2 = Number
)

func f(n Number) { println(n) }

func main() {
	var n int32 = 5
	f(&n)
}
```

Reply:
I think it's good idea

Reply:
@omarsy just found this: https://github.com/gnolang/gno/issues/1361 which confirms that we dont need to process NativeType